.github/workflows/bump_packages.yml

```
@@ -43,16 +43,28 @@ jobs:
         sudo curl --create-dirs -o /usr/local/bin/spectool https://pagure.io/rpmdevtools/raw/26a8abc746fba9c0b32eb899b96c92841a37855a/f/spectool.in
         echo 'echo "$GIT_AUTHOR_NAME <$GIT_AUTHOR_EMAIL>"' | sudo tee /usr/local/bin/rpmdev-packager
         sudo chmod +x /usr/local/bin/spectool /usr/local/bin/rpmdev-packager
+    - name: Install dependencies for nodejs
+      if: ${{ matrix.npm_name != null }}
+      run: |
+        sudo curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash - &&\
+        sudo apt-get install -y nodejs
+        sudo npm install npm2rpm
```

Wouldn't it be better to use https://github.com/actions/setup-node
```suggestion
    - name: Install nodejs
      if: ${{ matrix.npm_name != null }}
      uses: actions/setup-node@v3
      with:
        node-version: 16
    - name: Install npm2rpm
      if: ${{ matrix.npm_name != null }}
      run: npm install npm2rpm
```